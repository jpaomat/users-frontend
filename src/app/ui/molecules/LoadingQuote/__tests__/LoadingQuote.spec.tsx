import { render } from '@testing-library/react';
import { LoadingQuote } from '../LoadingQuote';

describe('<NavbarAuth /> component', () => {

	beforeEach(()=> jest.clearAllMocks);
	
	test('should render component', () => {
		const { container } = render(
			<LoadingQuote />
		);
		expect(container).toMatchSnapshot();
	});
});
