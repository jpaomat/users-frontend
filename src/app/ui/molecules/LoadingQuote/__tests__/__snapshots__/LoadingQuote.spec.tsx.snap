// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<NavbarAuth /> component should render component 1`] = `
<div>
  <div
    class="alert alert-info text-center"
  >
    Cargando ...
  </div>
</div>
`;
